2021-04-25 16:49:42,253 INFO    MainThread:46878 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 16:49:42,254 INFO    MainThread:46878 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 16:49:42,254 INFO    MainThread:46878 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_164942-1bsqxswh/logs/debug.log
2021-04-25 16:49:42,254 INFO    MainThread:46878 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_164942-1bsqxswh/logs/debug-internal.log
2021-04-25 16:49:42,255 INFO    MainThread:46878 [wandb_init.py:init():369] calling init triggers
2021-04-25 16:49:42,255 INFO    MainThread:46878 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 16:49:42,255 INFO    MainThread:46878 [wandb_init.py:init():418] starting backend
2021-04-25 16:49:42,255 INFO    MainThread:46878 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 16:49:42,270 INFO    MainThread:46878 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 16:49:42,277 INFO    MainThread:46878 [backend.py:ensure_launched():128] started backend process with pid: 46894
2021-04-25 16:49:42,278 INFO    MainThread:46878 [wandb_init.py:init():423] backend started and connected
2021-04-25 16:49:42,281 INFO    MainThread:46878 [wandb_init.py:init():463] updated telemetry
2021-04-25 16:49:42,282 INFO    MainThread:46878 [wandb_init.py:init():482] communicating current version
2021-04-25 16:49:42,952 INFO    MainThread:46878 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 16:49:42,953 INFO    MainThread:46878 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 16:49:43,137 INFO    MainThread:46878 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 16:49:43,202 INFO    MainThread:46878 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 16:49:43,203 INFO    MainThread:46878 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 16:49:43,203 INFO    MainThread:46878 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 16:49:43,207 INFO    MainThread:46878 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 16:49:43,207 INFO    MainThread:46878 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 16:49:50,196 INFO    MainThread:46878 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-25 16:49:50,197 INFO    MainThread:46878 [wandb_run.py:_restore():1466] restore
2021-04-25 16:49:52,393 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 16:49:52,821 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 16:49:53,159 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1949
}

2021-04-25 16:49:53,353 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1987
}

2021-04-25 16:49:53,456 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:53,558 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:53,660 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:53,763 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:53,865 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:53,968 INFO    MainThread:46878 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1987
  total_bytes: 1987
}

2021-04-25 16:49:55,098 INFO    MainThread:46878 [wandb_run.py:_show_files():1838] logging synced files
